Tests/XCUITests/HistoryTests.swift

```diff
@@ -91,6 +91,7 @@ class HistoryTests: BaseTestCase {
             waitForExistence(app.buttons["urlBar-cancel"], timeout: TIMEOUT_LONG)
         }
         navigator.performAction(Action.CloseURLBarOpen)
+        waitForNoExistence(app.buttons["urlBar-cancel"], timeoutValue: TIMEOUT)
```

Comment:
Let's ensure that the keyboard is closed before clicking the `settingsMenuButton`.

Note that when the keyboard is displayed, the `settingsMenuButton` is deemed exist and is hittable, but cannot be tapped. 🤷🏼‍♀️ Let's put a "guard" here before the test attempt to test the presence and tap on `settingsMenuButton.